tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+    "compilerOptions": {
+        "target": "es5",
+        "lib": [
+            "dom",
+            "dom.iterable",
+            "esnext"
+        ],
+        "allowJs": true,
+        "skipLibCheck": true,
+        "esModuleInterop": true,
+        "allowSyntheticDefaultImports": true,
+        "strict": true,
+        "forceConsistentCasingInFileNames": true,
+        "noFallthroughCasesInSwitch": true,
+        "module": "esnext",
+        "moduleResolution": "node",
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "noEmit": true,
+        "jsx": "react-jsx",
+        "noImplicitAny": false,
+        "typeRoots": [
+            "./src/types/global.d.ts",
+            "./node_modules/@types"
+        ]
+    },
+    "include": [
+        "src"
+    ]
```

Comment:
Cool work.
Can you support module resolution.

Reply:
it means alias. if u want to apply alias feature. we need to config the both(webpack and typescript). its a little bit complex. i think we can config simple first. just upgrade project day by day. i dont want to breaking change in project because if i change it, i dont make sure all of features are correct.

Reply:
👍 Sure no problem